core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala

```diff
@@ -55,6 +55,12 @@ trait StreamletContext {
    */
   def config: Config = streamletDefinition.config
 
+  /**
+   * The default bootstrapServers for the Kafka broker that has been installed or configured
+   * to be used globally for all Cloudflow applications
+   */
+  def defaultBootstrapServers = config.getString("cloudflow.kafka.bootstrap-servers")
```

Comment:
I think `internalKafkaBootstrapServers` or `cloudflowKafkaBootstrapServers` would be better options.

Reply:
Feel free to disagree

Reply:
I think that is better, thanks